gcloud/bigtable/client.py

```diff
@@ -86,12 +92,22 @@ class Client(_ClientFactoryMixin, _ClientProjectMixin):
                   interact with the Cluster Admin or Table Admin APIs. This
                   requires the :const:`ADMIN_SCOPE`. Defaults to :data:`False`.
 
+    :type user_agent: str
+    :param user_agent: (Optional) The user agent to be used with API request.
+                       Defaults to :const:`DEFAULT_USER_AGENT`.
+
+    :type timeout_seconds: int
+    :param timeout_seconds: Number of seconds for request time-out. If not
+                            passed, defaults to
+                            :const:`DEFAULT_TIMEOUT_SECONDS`.
+
```

Comment:
Most of it happens in gRPC, some of it happens in generated code as well.

We make a gRPC stub object:
https://github.com/dhermes/gcloud-python-bigtable/blob/21aa4a73df0fa7529285271e9fe2cc28b1e992e7/gcloud_bigtable/_helpers.py#L296-L299

The `stub_factory` is specific to the service. For example
https://github.com/dhermes/gcloud-python-bigtable/blob/21aa4a73df0fa7529285271e9fe2cc28b1e992e7/gcloud_bigtable/client.py#L55-L56
is a possible choice (`bigtable_table_service_pb2.early_adopter_create_BigtableTableService_stub`).